features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionView.kt

```diff
@@ -302,15 +250,3 @@ private fun ContentToPreview(state: VerifySelfSessionState) {
         goBack = {},
     )
 }
-
-private fun Modifier.shrinkableHeight(
-    min: Dp,
-    max: Dp,
-    minScreenHeight: Int = 720
-): Modifier = composed {
-    if (LocalConfiguration.current.screenHeightDp >= minScreenHeight) {
-        then(Modifier.height(max))
-    } else {
-        then(Modifier.height(min))
-    }
-}
```

Comment:
I also found that this was useless, because we are using weight and we are centering content, but maybe it was not. @jmartinesp any thoughts? I could move it to the design module if necessary. Let me know.
Screenshot test is still happy, but it maybe has some effect on other screen sizes?

Reply:
I think this was done to fix some weird issues in small displays, but I think those were already noticeable in the screenshots tests, so if we don't see them maybe it's fixed now. There were some changes to use a fixed number or emojis per row to make sure the content size was constant, so maybe that helped with this issue too.